10-inference-for-regression.Rmd

```diff
@@ -653,12 +671,12 @@ visualize(bootstrap_distn_slope) +
                             linetype = "dotted", color = "black")
 ```
 
-```{r bootstrap-distribution-slope-CI, echo=FALSE, fig.show='hold', fig.cap="Comparing three confidence intervals for the slope.", purl=FALSE, fig.height=2.5}
+```{r bootstrap-distribution-slope-CI, echo=FALSE, fig.show="hold", fig.cap="Comparing three confidence intervals for the slope.", purl=FALSE, fig.height=2.5}
 # Will need to make a tweak to the {infer} package so that it doesn't always display "Null" here
```

Comment:
Same note here as earlier. Can probably remove these commented lines.

Reply:
Removed the second block entirely here, since it became redundant with the correct titles